common/app/routes/Challenges/Show.jsx

```diff
@@ -42,12 +42,13 @@ const mapStateToProps = createSelector(
   challengeMetaSelector,
   paramsSelector,
   (
-    { dashedName, isTranslated },
+    { dashedName, isTranslated, guideUrl },
     { viewType, title },
     params
   ) => ({
     challenge: dashedName,
     isTranslated,
+    guideUrl,
```

Comment:
This is added to mapStateToProps but the value is not used anywhere in the component. This may be an unnecessary change.